packages/aws-lambda/src/triggers.js

```diff
@@ -52,6 +53,10 @@ exports.enrichSpanWithTriggerData = function enrichSpanWithTriggerData(event, co
   } else if (isInvokeFunction(context)) {
     span.data.lambda.trigger = 'aws:lambda.invoke';
     return;
+  } else if (isFunctionURLTrigger(event)) {
+    span.data.lambda.trigger = 'aws:lambda.functionurl';
```

Comment:
Sorry, I have one more tiny nitpick (which I should have brought up yesterday 🙈 ). The other triggers use `.` as a word separator. (`load.balancer`, `cloudwatch.logs` etc.)

For consistency, we could do the same here as well and go with `aws:lambda.function.url`. What do you think?

Reply:
That sounds like an excellent catch 🥇